### PR TITLE
advanced-usage: Mention `virtualpkg`.

### PR DESCRIPTION
**Pull request by flexibeast**
Partially addresses #201.

### CODE REVIEW

File: src/xbps/advanced-usage.md [hunk 1]
```diff
@@ -56,3 +56,14 @@ ignorepkg=sudo
 
 You will then be able to remove the `sudo` package using
 [xbps-remove(1)](https://man.voidlinux.org/xbps-remove.1).
+
+## Virtual Packages
+
+Virtual packages can be created with
+[xbps.d(5)](https://man.voidlinux.org/xbps.d.5) `virtualpkg` entries. For
+example, to make the `linux` package refer to the `linux5.6` package, create an
+xbps.d configuration file with the contents:
```
Comment (ericonr) [inline src/xbps/advanced-usage.md:65]: I think `xbps.d` should be inline code.

File: src/xbps/advanced-usage.md [hunk 2]
```diff
@@ -56,3 +56,14 @@ ignorepkg=sudo
 
 You will then be able to remove the `sudo` package using
 [xbps-remove(1)](https://man.voidlinux.org/xbps-remove.1).
+
+## Virtual Packages
+
+Virtual packages can be created with
+[xbps.d(5)](https://man.voidlinux.org/xbps.d.5) `virtualpkg` entries. For
+example, to make the `linux` package refer to the `linux5.6` package, create an
```
Comment (ericonr) [inline src/xbps/advanced-usage.md:64]: I believe "refer" isn't the best wording. I really like the man page explanation. It uses "resolve", in this case, but I would copy the whole phrase, really: "Any request to the virtual package will be resolved to the real package."

**Comment by flexibeast** [comment]
@ericonr: Thanks for the review; PR updated. Ok to merge?

**Comment by ericonr** [comment]
Merging. Thanks c:

**Pull request merged**